apps/ui-storybook-e2e/src/integration/dialog/dialog.cy.ts

```diff
@@ -129,8 +129,6 @@ describe('dialog--default', () => {
 				.findByText(/close nested dialog/i)
 				.click();
 
-			cy.wait(100);
```

Comment:
Because of this change is the e2e tests are failing. 
When not waiting for the closing the get('.cdk-overlay-backdrop') finds two occurances. 
I would check if the backdrop only has the length of 1, the cy waits for the closing of the dialog. 

`cy.get('.cdk-overlay-backdrop').should('have.length', 1);`